package.json

```diff
@@ -1,5 +1,12 @@
 {
-    "dependencies": {
-        "yuglify": "^0.1.4"
-    }
+  "scripts": {
+    "test": "eslint --ignore-pattern 'staticfiles/**/*.js' --ignore-pattern 'env/**/*.js' .; csslint --quiet --exclude-list='node_modules,staticfiles,env' --warnings=' ' ."
```

Comment:
`staticfiles/` is the output directory of `collectstatic` isn't it? We should `eslint` the `static/` files instead.